media/stylus/components/compat-tables/bc-browser.styl

```diff
@@ -0,0 +1,93 @@
+/*
+Icons in mobile view
+====================================================================== */
+
+@media $media-query-small-mobile {
+
+    [class*='bc-browser-']:before,
```

Comment:
Will the class always start with `bc-browser`? If so, we could consider using the `^=` selector to further reduce the likelihood of selecting an unrelated element.